test/parallel/test-util-internal.js

```diff
@@ -8,9 +8,24 @@ const fixtures = require('../common/fixtures');
 const {
   getHiddenValue,
   setHiddenValue,
-  arrow_message_private_symbol: kArrowMessagePrivateSymbolIndex
+  arrow_message_private_symbol: kArrowMessagePrivateSymbolIndex,
+  safeGetenvForJs
 } = process.binding('util');
 
+for (const oneEnv in process.env) {
+  console.log(oneEnv, safeGetenvForJs(oneEnv));
```

Comment:
Tests should, as a rule of thumb, be silent.  Don't print the inputs, the exception from `assert.strictEqual()` takes care of that (it's in the error message.)